src/main/java/racingcar/RacingCarGame.java

```diff
@@ -0,0 +1,30 @@
+package racingcar;
+
+import java.util.List;
+import java.util.Random;
+
+public class RacingCarGame {
+    private final List<Car> carList;
+    private final int tryCount;
+    private final ResultView resultView;
+
+    public RacingCarGame(List<Car> carList, int tryCount, ResultView resultView) {
+        this.carList = carList;
+        this.tryCount = tryCount;
+        this.resultView = resultView;
+    }
+
+    public void startGame(Random random) {
+        resultView.printResult();
+        for (int i = 0; i < tryCount; i++) {
+            moveCars(this.carList, random);
+            resultView.printCars(this.carList);
+        }
+    }
+
+    private void moveCars(List<Car> carList, Random random) {
+        for (Car car : carList) {
+            car.move(random);
```

Comment:
```suggestion
            car.move(random);
```

`Random`에 대한 개념이 너무 깊숙히 들어가지고  있는게 아닐까요?
`Car`라는 객체는 무엇이 궁금한것이고, 무엇만 알고 싶어할까요?
`car.move(random.nextInt..);`
- 단순 얼마인지가 궁금한게 아닐까요? 
- 그게 더 단순하고 일반화된 접근 아닐까요?
- 확장에 더 유리하지 않을까요?